scripts/assets/file_content_validation_checks.textproto

```diff
@@ -465,3 +465,13 @@ file_content_checks {
   exempted_file_name: "scripts/src/javatests/org/oppia/android/scripts/regex/RegexPatternValidationCheckTest.kt"
   exempted_file_name: "utility/src/main/java/org/oppia/android/util/parser/image/UrlImageParser.kt"
 }
+file_content_checks {
+  filename_regex: ".+.xml"
+  prohibited_content_regex: "^\s*<meta-data\s+android:name\s*=\s*"firebase_analytics_collection_deactivated"\s*android:value\s*=\s*"false".+$"
```

Comment:
@chrislee115 PTAL at the "Files Changed" tab on GitHub (https://github.com/oppia/oppia-android/pull/4944/files). Two questions:

- Do you need a double backslash here?
- Please add a newline at EOF.

Additionally, in your PR description, please describe how you validated the change and show screenshots as proof that it works correctly. Thanks!

Reply:
Adding a newline EOF.
What is the double backslash are you referring to?
Also is there a preferred way to test scrips/assets? I'm not sure if I particularly saw anything relevant in the testing guide, but I may have missed something?

Reply:
If you look at the other regexes in the file, you'll see that they use a double backslash `\\` instead of a single one `\`, and that the single ones are highlighted by github in red. This might indicate that the backslash here is escaping the `s` rather than being treated as a literal backslash. You'll probably want to double-check what works through manual testing.

For (manual) testing, I'd suggest that you create a file with prohibited content, run the script, and check that it does the right thing. Show this proof in your PR description, so that it is possible to validate that your changes actually fix the issue.